### PR TITLE
Fixes missing floor tile stack icons, and adds new tile stack sprites.

### LINKED ISSUE #2524: Missing tile sprites
BUG: dark grey perforated tiles, and dark techfloor tiles are invisible when plied off the plating.

### PR DESCRIPTION
**Pull request by Kurgis**
Fixes  #2524
Certain tiles will no longer appear invisible, and will use new sprites.
Re-names "tur carpet" to "turquoise carpet", "carpet" to "red carpet".
Moves tile sprites from the items folder into a sub-folder for stack items.
Adds various tile stack sprites.

pr thing by 2

### CODE REVIEW

**Comment by NanakoAC** [comment]
As mentioned in discord, the naming of the tiles as light, normal, dark was an intentional design and shouldn't be changed, please revert naming changes

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 1]
```diff
@@ -150,14 +172,17 @@
 /obj/item/stack/tile/floor/steel/brown_perforated
 	name = "steel brown perforated tile"
 	singular_name = "steel brown perforated tile"
+	icon_state = "tile_steel_brownperforated"
 
-/obj/item/stack/tile/floor/steel/gray_perforated
+/obj/item/stack/tile/floor/steel/gray_perforated //TODO: Turn all references of "gray_perforated" to "grey_perforated"
```
Comment (NanakoAC) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: this naming change is fine

File: code/game/turfs/flooring/flooring.dm [hunk 2]
```diff
@@ -345,7 +345,7 @@ var/list/flooring_types
 	build_type = /obj/item/stack/tile/carpet/blucarpet
 
 /decl/flooring/carpet/turcarpet
-	name = "tur carpet"
+	name = "turqoise carpet"
```
Comment (NanakoAC) [inline code/game/turfs/flooring/flooring.dm]: turquoise

**Comment by Kurgis** [comment]
No naming changes have been performed besides naming "tur carpet" to "turquoise carpet". I see the spelling mistake, and it will be corrected.

**Comment by Kurgis** [comment]
Ready for review.

**Pull request merged**